tests/Microsoft.Identity.Test.Integration.netcore/Microsoft.Identity.Test.Integration.NetCore.csproj

```diff
@@ -33,5 +33,11 @@
 
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="..\Microsoft.Identity.Test.Integration.NetFx\federationMetadata.xml">
```

Comment:
Yeah, adding file resources to the projects still requires knowledge of msbuild.... I wish VS would provide a better solution.

Reply:
same